src/Storage/Controllers/InstancesController.cs

```diff
@@ -800,5 +802,19 @@ private string BuildRequestLink(string continuationToken = null, Dictionary<stri
                 return selfUrl;
             }
         }
+
+        /// <summary>
+        /// Validates app id
+        /// </summary>
```

Comment:
No need for comments on private methods. Especially those that don't add anything.

Or change the comment to for example:
Ensures that the app value contains a '/' and that there are no newline characters.